src/main/java/Board.java

```diff
@@ -69,6 +69,9 @@ public Player getCurrentPlayer() {
         return null;
     }
 
+    public Piece[][] getBoard() {
```

Comment:
There is no need to expose implementation details. The Board class supplies an interface for manipulation of its structure through `getPieceAtAbsCoords()` and `setPieceAtAbsCoords()`.